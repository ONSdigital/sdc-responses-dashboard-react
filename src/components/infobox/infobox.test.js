import React from 'react';
import ReactDOM from 'react-dom';
import InfoPanel from './index';


let props = {
    type: "downloads",
    number: 400
}


it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<InfoPanel {...props}/>, div);
  ReactDOM.unmountComponentAtNode(div);
});
