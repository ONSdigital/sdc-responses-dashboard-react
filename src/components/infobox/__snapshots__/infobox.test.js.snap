// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoPanel renders without crashing 1`] = `
<styled.div>
  <styled.div>
     
    400
     
  </styled.div>
  <styled.div>
     
  </styled.div>
</styled.div>
`;
